### DISCUSSION TITLE
GigaGAN / AuraSR support

### DISCUSSION THREAD

**Discussion by Belluxx** [chaiNNer-org/chaiNNer#2980] [Q&A] [answered]
I am getting an error when loading AuraSR PyTorch ckpt. Is it currently unsupported? The architecture should be GigaGAN.

**Comment by joeyballentine** [comment, marked as answer]
It's unsupported yes. Check the list of supported arches in the readme. We can add it, but so far everyone who has used it has said it's awful...

**Reply by Belluxx** [reply]
Got it. Thanks for the infos!
